reports/Indiana.md

```diff
@@ -14,6 +14,21 @@ id: in-fortwayne-1
 * https://twitter.com/_isabel_a/status/1266580681251000321
 
 
+### Police tear gas journalist | May 29th
+
+Karli VanCleave, a local journalist, was hit in the back with a tear gas canister, while she says she was standing in the middle of a crowd of journalists, carrying equipment and wearing clothing marked with her outlet's logo. She claims that police were firing tear gas indiscriminately, without warning, in all directions.
+
+tags: journalist, tear-gas, tear-gas-canister
+
+id: in-fortwayne-6
+
+**Links**
+
+* https://pressfreedomtracker.us/all-incidents/wpta-journalist-hit-tear-gas-canister-while-covering-indiana-protest/
```

Comment:
I don't love that there's no direct evidence, but I'm inclined to still include it. 

But I'm a little surprised that with two reporters talking about it, there's no visual evidence